2020 announcement blog post (#99)
RustFestEU/blog.rustfest.eu: _posts/2020-06-06-announcing-rustfest-2020.md

---
layout: post
title: "Announcing RustFest Global/Online 2020 🎉"
description: "Making RustFest Global for November 2020 in times of the Coronavirus"
socialImageSrc: "/assets/posts/2020-global/2020-06-06_announcement.png"
authors:
  - dns2utf8
---

> TLDR: The next RustFest will be on the weekend of ???2020-11-07 and 2020-11-08??? and it will be a online community conference 🎉

Reviewer: "an online community"

Author: Thanks :)

# Future<Output = RustFest::Global2020>

So how will it work?

## CfP for talks

The Call for Participation will be similar to the ones in the previous years, where we will start with a blind phase followed by a second round when a part of the team will see your names.

> Please note that dropping your name will not help your talk to make the cut.

Reviewer: Doesn't it even disqualify? (not sure)

Author: It used to, but we have not finalized that yet.

### Technical details

Over the next weeks we will test different setups.

## CfP for workshops

> We are looking into how to make workshops reliable and fun for everyone.

Reviewer: reliable, fun and accessible for everyone.

Author: :+1:

Currently, we are looking into good formats.

So we will have a second 

## What are we looking for?

> ## What to do meanwhile?

Reviewer: Instead of "meanwhile", use "in the meantime"

Author: Thanks :)

---
layout: post

> title: "Announcing RustFest Global/Online 2020 🎉"

Reviewer 2: ```suggestion
title: "Announcing RustFest Global 2020 🎉"
```

description: "Making RustFest Global for November 2020 in times of the Coronavirus"
socialImageSrc: "/assets/posts/2020-global/2020-06-06_announcement.png"
authors:
  - dns2utf8
---

> TLDR: The next RustFest will be on the weekend of 2020-11-07 and 2020-11-08 and it will be an online community conference 🎉

Reviewer 2: ```suggestion
The summary: The next RustFest will be on the weekend of 7th & 8th November (2020-11-07 and 2020-11-08) and it will be an online community conference! 🎉
```

I would avoid abbrevations.
I also think that spelling out the month is a good idea.

# Future<Output = RustFest::Global2020>

> 

Reviewer 2: We should give a bit of an introduction:

```suggestion

RustFest 2020 was planned to happen about now, in Utrecht, the Netherlands. 
As the COVID-19 pandemic spread around the world and forced us to shelter at home we had to put a stop to planning a large in-person event.
The team still wanted to make something happen and we talked through several options.

We are now very happy to announce that RustFest Global 2020 will happen as an online community conference on the weekend of 7th & 8th November (2020-11-07 and 2020-11-08)! 🎉
```

Writing it out like this I think we might even drop the tldr/summary above?

So, how will it work?

> ## CfP for talks

Reviewer 2: ```suggestion
## Talks & Call for Proposals
```

> 

Reviewer 2: ```suggestion

RustFest Global will feature talks across a wide range of topics, just the same as our in-person events in the past had.
We will stream all talks and provide a way to interact with the speakers & other attendees. Of course the talks will also be recorded and be available after the conference.
```

> The Call for Participation will be similar to the ones in the previous years, where we will start with a blind phase followed by a second round when a part of the team will see your names.

Reviewer: "in which a part of the team will see your names"

> The Call for Participation will be similar to the ones in the previous years, where we will start with a blind phase followed by a second round in which a part of the team will see your names.

Reviewer 2: ```suggestion
The Call for Participation will be similar to the ones in the previous years, where we will start with a blind phase followed by a second round, where we do de-anonymise submissions to find a mix of seasoned and new speakers.
Voting will happen by a committee of RustFest organizers and community members.
```

> Please note that dropping your name will not help your talk to make the cut.

Reviewer 2: We can just leave this out. We will post these details when we announce the CfP.

```suggestion
We want you to speak at RustFest and explicitly encourage new speakers to talk about their projects, ideas & experiences.
We will provide further details on the CfP process in the coming weeks.
```

### Technical details

> Over the next weeks we will test different setups.

Reviewer 2: ```suggestion
We haven't settled on the exact details yet, but will test different setups over the next weeks.
All talks will be streamed and recorded and we will help with the technical setup.
You can give the talk from the coziness of your home, no travel required.
```

## CfP for workshops

> We are looking into how to make workshops reliable, accessible and fun for everyone.

Reviewer 2: ```suggestion
Workshops have always been part of RustFest.
Bringing them online will be a harder task, but we will try to do just that.

We are looking into how to make workshops reliable, accessible and fun for everyone.
We might provide additional training for coaches and help with planning as best as we can.
```

Currently, we are testing good formats with the [Zürich](https://www.meetup.com/Rust-Zurich/) and [Berlin](https://berline.rs/) meetups.

> So we will have a second CfP for the workshops once we decide on a specific format.

Reviewer 2: ```suggestion
So we will have a second CfP for the workshops once we decide on a specific format.
If you already have an idea for a workshop format or content that can work online, please approach us via email: [team@rustfest.eu](mailto:team@rustfest.eu).
```

## What are we looking for?

> This is not the final list, it should give you some pointers:

Reviewer 2: ```suggestion
You are free to talk about anything Rust-related, but here's an incomplete list of relevant topics:
```

> * How to stay connected in your part of the rust community?

Reviewer 2: ```suggestion
* How to stay connected in your part of the Rust community?
```

    * Do you reach out to new people?
* How do you learn about useful crates?

> * Show off your cool feature/crate/contribution?

Reviewer 2: ```suggestion
* Show off your cool feature/crate/contribution
```

> * Are you able to integrate rust in your day job? Or did you experience a misfits? 

Reviewer: Or did you experience a misfit / misfits

Author: Do you suggest to add this as a subpoint?

> * Are you able to integrate rust in your day job? Or did you experience a misfits? 

Reviewer 2: ```suggestion
* Are you able to integrate Rust in your day job? Or did you experience a misfits? 
```

> Or to make it even shorter: What have you learned (with rust)?

Reviewer 2: ```suggestion
Or to make it even shorter: What have you learned (with Rust)?
```

 

## What to do in the meantime?

> With the global lock down: Attend online meetups.

Reviewer 2: It's not a lock down everywhere anymore.

```suggestion
While in-person events are still not possible: Attend online meetups.
```

> As mentioned before [Berlin](https://berline.rs/) and [Zürich](https://www.meetup.com/Rust-Zurich/) and many others are open to anybody on the planet to talk about a subject.

Reviewer 2: ```suggestion
As mentioned before [Berlin](https://berline.rs/) and [Zürich](https://www.meetup.com/Rust-Zurich/) are running online now and are open to anybody on the planet to join in and talk about Rust. 
Other meetups & events are also moving online. 
```

Author: I incooperated this, thanks :)

### Speaker preparations

If you plan to submit a talk or a workshop for this or any other conference, why not record yourself talking about something for 5 minutes.

This will allow you to see how the recording quality is.
Pro Tip: If you want to invest in something, start with brighter lights. 

> Buying any LED bulb (usually with the E27 fitting) with more than 1000 lumen and a colour of 2500 to 2700 Kelvin for less than 20€ will have more effect than a fancy webcam way more money.

Reviewer: than a fancy webcam that costs way more money

Author: Thank you :)

> Buying any LED bulb (usually with the E27 fitting) with more than 1000 lumen and a colour of 2500 to 2700 Kelvin for less than 20€ will have more effect than a fancy webcam that costs way more money.

Reviewer 2: We might not need to be that specific.

```suggestion
Buying any LED bulb with more than 1000 lumen and a warm colour will probably improve quality more than a fancy webcam will.
```

Author: Incorporated this. Thanks :+1:

Plus you can use the bulb as a normal light too.

<!--
<div class="card">
    <div class="imgbox" data-frame="/assets/rf-rainbow-ferris.png">
      <img src="/assets/rf-rainbow-ferris.png" alt="RustFest Logo" class="frame">
    </div>

    <section class="infobox" spellcheck="false" contenteditable="">

>       <span class="twitter">Save the date!</span>

Reviewer 2: ```suggestion
      <span class="twitter">Save the date: November 7th & 8th</span>
```

      <span class="speaker" style="margin-right: -10px;font-size: 27px;">RustFest Global</span>

>       <span class="title">Get ready to submit a talk! 🎉</span>

Reviewer 2: No CfP yet, so how about:
```suggestion
      <span class="title">An online community conference! 🎉</span>
```
